app/components/profilefield.jsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import { useUserContext } from '../context/usercontext';
+import TextField from '@mui/material/TextField';
+import {useState} from 'react'
+
+export default function ProfileField({field}) {
+    const {userData, setUserData, user} = useUserContext();
+    const [edit, setEdit] = useState(false);
+    // console.log(userData)
+    let text;
+    switch (field) {
+        case 'First name': 
+            text = userData.first;
+            break;
+        case 'Last name':
+            text = userData.last; 
+            break;
+        case 'Gender':
+            text = userData.gender;
+            break;
+        case 'Date of birth':
+            text = userData.date;
+            break;
+        default:
+            throw(Error(`unknown profile field props: ${field}`))
+    }
+    console.log(text)
```

Comment:
we should remove console logs when we're ready to merge stuff into the main branch.